Dockerfile

```diff
@@ -1,6 +1,14 @@
 # ===============================================
-FROM helsinkitest/node:20-slim as appbase
+FROM registry.access.redhat.com/ubi9/nodejs-18 AS appbase
```

Comment:
Why downgrade node from 20 to 18?

Reply:
The Ubi-image is the preferred image base and since node 20 ubi-image is not available yet, we have been using 18 in other projects as well.

Reply:
...and since there were some permission issues, I changed the image (to the preferred), because I knew the configuration that would work 😄

Reply:
Ok, had to test myself to see whether they really aren't serving nodejs-20, and so it seems:
```bash
#4 [app internal] load metadata for registry.access.redhat.com/ubi9/nodejs-20:latest
#4 ERROR: pulling from host registry.access.redhat.com failed with status code [manifests latest]: 401 Unauthorized
```
Too new for ubi9, hmm, looking at https://nodejs.org/en/blog/announcements/v20-release-announce Node.js 20 was released on 2023-04-18 so its ~6 months old now. Well, interesting anyway.